### DISCUSSION TITLE
Go-e without cloud

### DISCUSSION THREAD

**Discussion by rolandovoces** [cathiele/homeassistant-goecharger#85] [Q&A]
Hoi Carsten or someone who can help me,
the cloud of go-e is not reliable and therefor I have problems with some automations. I activated the local mqtt broker but it seem that I made a mistake or something else..
I activated the mqtt with the app ( ich habe ich eine alte go-e)
![image](https://github.com/cathiele/homeassistant-goecharger/assets/110081957/3a146b44-c8e1-4170-87b7-bc13757616c6)

and

![image](https://github.com/cathiele/homeassistant-goecharger/assets/110081957/aec9a531-a383-40b3-9553-4a96f29416d9)

and

<img width="404" alt="image" src="https://github.com/cathiele/homeassistant-goecharger/assets/110081957/92ae9424-596b-4ed2-b9a9-625c0747b312">

and the error (cloud is down)

<img width="849" alt="image" src="https://github.com/cathiele/homeassistant-goecharger/assets/110081957/3c473911-31a5-457e-928f-39390519a419">

**What I have to change that the local access via the local mqtt broker works?**

Many thanks in advance

**Comment by maccowley** [comment]
Where did you get the screen with the broker options from?
To use this integration you just need to activate the API v1 and enter the IP address of the charger. 
![IMG_0130](https://github.com/cathiele/homeassistant-goecharger/assets/109413785/41bfb77e-fa66-4c6e-8cec-6ab643940805)

**Reply by rolandovoces** [reply]
Hi, yesterday I got an hint from the go-e support. It seems that the wi-fi module is broken. The signal is weak. I am still in contact for solutions. 
Many thanks for your help.
